pkgs/applications/science/electronics/fparser/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "fparser";
+  version = "4.5.1";
```

Comment:
how is this determined? upstream doesn't have any releases
https://github.com/thliebig/fparser/releases

Reply:
It's buried a bit. See [here](https://github.com/thliebig/fparser/blob/a59e1f51e32096bfe2a0a2640d5dffc7ae6ba37b/docs/fparser.html#L10).

Reply:
Should probably be `2015-09-25` since that's the date of the revision you're pinning to. I don't know if it needs an `unstable-` prefix -- @jonringer?

Reply:
if taking from a revision, the current pratice is take the commit date, so `version = unstable-2015-09-25` in this case